src/account/utils/signature.cairo

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo vX.Y.Z (account/utils/signature.cairo)
+
+use ecdsa::check_ecdsa_signature;
+use openzeppelin::account::eth_account::interface::EthPublicKey;
+use openzeppelin::account::utils::secp256k1::Secp256k1PointPartialEq;
+use starknet::eth_signature::Signature;
+use starknet::secp256_trait::{is_signature_entry_valid, recover_public_key};
+use starknet::secp256k1::Secp256k1Point;
+
+fn is_valid_signature(msg_hash: felt252, public_key: felt252, signature: Span<felt252>) -> bool {
```

Comment:
now that this is a helper, should we be explicit about the sig scheme (stark curve)?

Reply:
I changed it to `is_valid_stark_signature`, let me know what you think.